db/entity/DiscordServerConfig.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  DeleteDateColumn,
+  Index,
+  Relation,
+} from "typeorm";
+import type { StarkyModuleConfig } from "../../starkyModules/types";
+import { DiscordServer } from "./DiscordServer";
+
+@Entity()
+@Index(["id", "DiscordServerId", "deletedAt"], {
+  unique: true,
+  where: '"deletedAt" IS NOT NULL',
+})
+@Index(["id", "DiscordServerId"], {
+  unique: true,
+  where: '"deletedAt" IS NULL',
+})
+export class DiscordServerConfig {
+  @PrimaryGeneratedColumn("increment")
+  id: string;
+
+  @Column()
+  discordServerId: string;
+
+  @Column()
+  starknetNetwork: string;
+
+  @Column()
+  discordRoleId: string;
+
+  @Column()
+  starkyModuleType: string;
+
+  @Column("jsonb", { nullable: false, default: {} })
+  starkyModuleConfig: StarkyModuleConfig;
+
+  @ManyToOne((type) => DiscordServer, (server) => server.serverConfigs)
+  discordServer: DiscordServer;
+
+  @DeleteDateColumn()
+  deletedAt?: Date;
```

Comment:
no sure why you need `deletedAt` and `softRemove`, would using `remove" work in [deleteConfigCommand.ts](https://github.com/nmalzieu/starky/pull/1/files#diff-bac18a2b9b3ba1aa360e8367c83fc95e7a5274cdb2dff9533410738d9863066d) ?

Reply:
We don't really need it, but i thought maybe we'd like to keep a trace after deleting